hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -1163,7 +1166,9 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
                 taker_trading_pair, True, quote_asset_amount
             ).result_price
             adjusted_taker_price = taker_price * taker_slippage_adjustment_factor
-            order_size_limit = min(base_asset_amount, quote_asset_amount / adjusted_taker_price)
+            order_size_limit = base_asset_amount
```

Comment:
I think setting order_size_limit here as base_asset_amount will remove the intention of this method. This method is to check for sufficient balance on both the maker and taker account. By ignoring taker balance due to no taker price, we are not checking for sufficient taker balance.
As we already have the order price in this function, it may be better if we use the order price directly on line 1165 to avoid this error.